inspirehep/modules/records/api.py

```diff
@@ -459,7 +460,8 @@ def _download_to_docs_or_figs(
         if key not in self.files:
             key = self._get_unique_files_key(base_file_name=key)
 
-        stream = fsopen(doc_or_fig_obj['url'], mode='rb')
+        url = unquote(doc_or_fig_obj['url'])
```

Comment:
is it clear that you're dealing with a local file here? otherwise unquoting is wrong.